### PR TITLE
fix(curriculum): remove comments and extra code

### LINKED ISSUE #48527: Overly strict comments
### Describe the Issue

The declaration of arr doesn’t need to be guarded by ‘Do Not Change’ comments. There exist many solutions (3 in the guide) that do not need that variable declaration.

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/wherefore-art-thou

### Expected behavior

These comments should be removed or at least moved.

### System

All

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by bryanleemoore**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #48527

### CODE REVIEW

**Review by Sboonny — APPROVED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @bryanleemoore, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/wherefore-art-thou.md [hunk 1]
```diff
@@ -112,12 +112,7 @@ assert.deepEqual(
 
 ```js
 function whatIsInAName(collection, source) {
```
Comment (jeremylt) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/wherefore-art-thou.md:114]:
```suggestion
function whatIsInAName(collection, source) {

```
I think that usually we leave a blank line here

Comment (naomi-lgbt) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/wherefore-art-thou.md:114]: Looks like the blank line is on 115 - the diff on this one just isn't great.

Comment (jeremylt) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/wherefore-art-thou.md:114]: Oh, bad GitHub

**Review by jeremylt — APPROVED** [review]

**Comment by naomi-lgbt** [comment]
Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**